### PR TITLE
web: importing/exporting feature update

### PR DESCRIPTION
**Pull request by tinyjin**
Feature update for ThorVG Viewer's requirements.
- new loading feature - tvg, svg, jpg, png
- new exporting feature - tvg

Issue: #1871

### CODE REVIEW

File: src/bindings/wasm/tvgWasmLottieAnimation.cpp [hunk 1]
```diff
@@ -182,27 +187,23 @@ class __attribute__((visibility("default"))) TvgLottieAnimation
...
     // Saver methods
-    bool save2Gif(string data, string mimetype, int width, int height, int fps)
+    bool save(string mimetype)
     {
-        errorMsg = NoError;
+        if (mimetype == "gif") {
+            return save2Gif();
+        } else if (mimetype == "tvg") {
+            return save2Tvg();
+        }
```
Comment (hermet) [inline src/bindings/wasm/tvgWasmLottieAnimation.cpp:196]: please add "png" as well.

**Review by hermet — COMMENTED** [review]
1. Please consider the png exporting as well.
2. Have you tested the gif exporting?

File: src/bindings/wasm/tvgWasmLottieAnimation.cpp (hunk 1 above)
Comment (tinyjin) [reply src/bindings/wasm/tvgWasmLottieAnimation.cpp:196]: > Please consider the png exporting as well.

ignored as we decided to build it on viewer

**Comment by tinyjin** [comment]
> 2. Have you tested the gif exporting?

@hermet Yes, I have and see it works

**Review by hermet — APPROVED** [review]

**Pull request merged**